src/panels/lovelace/cards/hui-entity-filter-card.js

```diff
@@ -77,12 +77,20 @@ class HuiEntitiesCard extends PolymerElement {
     element.hass = hass;
   }
 
+  _computeVisibility(entitiesLength) {
+    if (this._config.hide_if_empty) {
```

Comment:
This is incorrect. This means that if I change the config, and set hide_if_empty to false, it might never show itself again.

Reply:
I'm also not a big fan of negative config options that then have to be set to true.

I prefer: `show_empty: true/false` with default `true`.